DataFormats/ForwardDetId/src/classes_def.xml

```diff
@@ -30,8 +30,15 @@
   <class name="BTLDetId" ClassVersion="3">
    <version ClassVersion="3" checksum="1515591716"/>
   </class>
-  <class name="ETLDetId" ClassVersion="3">
+  <class name="ETLDetId" ClassVersion="4">
+   <version ClassVersion="4" checksum="1644731879"/>
    <version ClassVersion="3" checksum="1644731879"/>
+   <ioread sourceClass = "ETLDetId" version="[-4]" targetClass="ETLDetId" source="" target="">
```

Comment:
@icosivi @casarsa most likely the version here should be [-3], we want the conversion on older ClassVersion data. This would explain why everything looks ok while writing, but then it miserably fails after reading back data from file.

Reply:
while it looks ok reading data from previous versions...